src/settings.ts

```diff
@@ -28,6 +28,7 @@ export const SPARKLEVERSE_TERMS_AND_CONDITIONS_URL =
   "https://sparklever.se/terms-of-use/";
 export const SPARKLEVERSE_PRIVACY_POLICY =
   "https://sparklever.se/privacy-policy/";
+export const SPARKLEVERSE_ITY = "https://sparklever.se/ity/";
```

Comment:
Maybe rename to SPARKLEVERSEITY_URL or anything that ends in `_URL`

Reply:
@goran-peoski-work fixed in e4059b28059129a2c241deb31894acd1d0feac1a